file/src/main/java/org/apache/zeppelin/file/HDFSFileInterpreter.java

```diff
@@ -311,7 +311,7 @@ public boolean isDirectory(String path) {
                 String beforeLastPeriod = unfinished.substring(0, unfinished.lastIndexOf('.') + 1);
                 //beforeLastPeriod should be the start of fs.pathSuffix, so take the end of it.
                 String suggestedFinish = fs.pathSuffix.substring(beforeLastPeriod.length());
-                suggestions.add(suggestedFinish);
+                suggestions.add(new InterpreterCompletion(suggestedFinish,suggestedFinish));
```

Comment:
ci failed ',' character between ' '  white space.
You need to add a white space.

maybe
```
suggestions.add(new InterpreterCompletion(suggestedFinish,suggestedFinish));
```
to
```
suggestions.add(new InterpreterCompletion(suggestedFinish, suggestedFinish));
```